src/stories/card.stories.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styled from 'styled-components';
+import { action } from '@storybook/addon-actions';
+
+export default { title: 'Card' };
+
+const CardContainer = styled.div`
```

Comment:
We probably will prefer to import the **real** component and not to create a new one

```js
import Card from '../me/components/Card';
```